src/graph_config_dialog.js

```diff
@@ -326,25 +337,23 @@ export function GraphConfigurationDialog(dialog, onSave) {
             dividerAfter: true
         });
 
-        for (i = 0; i < exampleGraphs.length; i++) {
-            var
-                graph = exampleGraphs[i],
-                li = $('<li><a href="#"></a></li>');
+        for (let i=0; i < exampleGraphs.length; ++i) {
```

Comment:
Same as above

Reply:
I do not know about JS, but in C  "++i" is more optimal code then "i++"
The spaces - Ok.

Reply:
i++ is the postfix increment operator, which increments the value of i after it is used in an expression. On the other hand, ++i is the prefix increment operator, which increments the value of i before it is used in an expression.

Reply:
> i++ is the postfix increment operator, which increments the value of i after it is used in an expression. On the other hand, ++i is the prefix increment operator, which increments the value of i before it is used in an expression.

i++ create tmp variable to store values, the ++i - is not.
In these cases it is not need to use unincrement value.
But for one code style - i've made change.